### ISSUE TITLE
Crash while viewing tutorial

### ISSUE THREAD

**Issue by bheimbaugh** [anegostudios/VintageStory-Issues#3259]
### Game Version

v1.18.15

### Platform

Windows

### Modded

None

### SP/MP

Singleplayer

### Description

[Win11] Crash when using [w][a][s][d] keys to move while viewing "First steps" tutorial.

### How to reproduce

First time install on Win11. Created first world (Exploration). 

- Open Handbook
- Start "first steps" tutorial
- First page of tutorial describes movement keys. Use [w][a][s][d] keys to move with tutorial open. 
- Crash (repeated 4 times without fail). 

### Logs

```Running on 64 bit Windows with 31904 MB RAM
Game Version: v1.18.15 (Stable)
11/27/2023 12:57:51 PM: Critical error occurred
Loaded Mods: game@1.18.15, creative@1.18.15, survival@1.18.15
System.InvalidOperationException: Stack empty.
   at System.Collections.Generic.Stack`1.ThrowForEmptyStack()
   at System.Collections.Generic.Stack`1.Peek()
   at Vintagestory.GameContent.GuiDialogHandbook.ReloadPage() in VSSurvivalMod\Systems\Handbook\Gui\GuiDialogHandbook.cs:line 256
   at Vintagestory.GameContent.ModSystemTutorial.reloadTutorialPage() in VSSurvivalMod\Systems\Handbook\Tutorial\ModSystemTutorial.cs:line 267
   at Vintagestory.GameContent.ModSystemTutorial.onStateUpdate(ActionBoolReturn`1 stepCall) in VSSurvivalMod\Systems\Handbook\Tutorial\ModSystemTutorial.cs:line 238
   at Vintagestory.GameContent.ModSystemTutorial.Input_InWorldAction(EnumEntityAction action, Boolean on, EnumHandling& handled) in VSSurvivalMod\Systems\Handbook\Tutorial\ModSystemTutorial.cs:line 114
   at Vintagestory.Client.NoObf.InputAPI.TriggerInWorldAction(EnumEntityAction action, Boolean on, EnumHandling& handling) in VintagestoryLib\Client\API\InputAPI.cs:line 37
   at Vintagestory.API.Common.EntityControls.AttemptToggleAction(EnumEntityAction action, Boolean on) in VintagestoryApi\Common\Entity\EntityControls.cs:line 337
   at Vintagestory.Client.NoObf.SystemPlayerControl.OnGameTick(Single dt) in VintagestoryLib\Client\Systems\Player\PlayerControl.cs:line 68
   at Vintagestory.Common.EventManager.TriggerGameTick(Int64 ellapsedMilliseconds, IWorldAccessor world) in VintagestoryLib\Common\EventManager.cs:line 59
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt) in VintagestoryLib\Client\ClientMain.cs:line 796
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime) in VintagestoryLib\Client\ClientMain.cs:line 712
   at Vintagestory.Client.GuiScreenRunningGame.RenderToPrimary(Single dt) in VintagestoryLib\Client\MainMenu\Screens\GuiScreenRunningGame.cs:line 200
   at Vintagestory.Client.ScreenManager.Render(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 668
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 643
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 77
   at OpenTK.Windowing.Desktop.GameWindow.Run()
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs) in VintagestoryLib\Client\ClientProgram.cs:line 313
   at Vintagestory.Client.ClientProgram.<>c__DisplayClass9_0.<.ctor>b__1() in VintagestoryLib\Client\ClientProgram.cs:line 129
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93

Event Log entries containing Vintagestory.exe, the latest 3
==================================
{ TimeGenerated = 11/27/2023 12:55:55 PM, Site = , Source = Windows Error Reporting, Message = Fault bucket 2007150627803250050, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Vintagestory.exe
P2: 1.18.15.0
P3: 64e841c5
P4: openal32.dll
P5: 1.23.0.0
P6: 63dd31ad
P7: 40000015
P8: 00000000000df046
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.ff6392dd-24e4-4ce4-ab7e-c7056c12d162.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.25b9f835-147d-46cf-bd56-5707862baeaf.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.30a56fc1-9efa-47ea-b6e6-35cc9c3bb95c.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.4c42b066-1fa0-4bb9-9a79-bc3058b64fd3.tmp.txt
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.b4bf659a-977d-443c-84af-4b44dabf2883.tmp.xml

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Vintagestory.exe_3278c09e339ad58fdb4a48929ddf95e5f2060ca_ee8069fc_4d215cdc-c160-4fd7-b1c1-4761e815ec9b

Analysis symbol: 
Rechecking for solution: 0
Report Id: 43b432be-0a3f-4b91-ac60-5f15643388ff
Report Status: 268566528
Hashed bucket: 38dd2bbe130a69d3fbdad4dc94ea3d82
Cab Guid: 0 }
--------------
{ TimeGenerated = 11/27/2023 12:55:54 PM, Site = , Source = Application Error, Message = Faulting application name: Vintagestory.exe, version: 1.18.15.0, time stamp: 0x64e841c5
Faulting module name: openal32.dll, version: 1.23.0.0, time stamp: 0x63dd31ad
Exception code: 0x40000015
Fault offset: 0x00000000000df046
Faulting process id: 0x0x3bf8
Faulting application start time: 0x0x1da2163372ba3f9
Faulting application path: C:\Users\brook\AppData\Roaming\Vintagestory\Vintagestory.exe
Faulting module path: C:\Users\brook\AppData\Roaming\Vintagestory\Lib\openal32.dll
Report Id: 43b432be-0a3f-4b91-ac60-5f15643388ff
Faulting package full name: 
Faulting package-relative application ID:  }
--------------
{ TimeGenerated = 11/27/2023 12:53:21 PM, Site = , Source = Windows Error Reporting, Message = Fault bucket 2007150627803250050, type 4
Event Name: APPCRASH
Response: Not available
Cab Id: 0

Problem signature:
P1: Vintagestory.exe
P2: 1.18.15.0
P3: 64e841c5
P4: openal32.dll
P5: 1.23.0.0
P6: 63dd31ad
P7: 40000015
P8: 00000000000df046
P9: 
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.0a64d1b7-b5b8-43f1-aea9-d92184ec8284.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.83975703-42d4-42ac-b80e-405545be7608.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.36c53e40-80a5-454e-957a-7714de8ecda1.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.251ea343-d67a-4047-8579-98858208e13d.tmp.txt
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER.a0e23d22-e8be-45f6-bee8-e67df50d4be8.tmp.xml

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Vintagestory.exe_3278c09e339ad58fdb4a48929ddf95e5f2060ca_ee8069fc_f2e46107-e245-4454-8340-26c543f52d6c

Analysis symbol: 
Rechecking for solution: 0
Report Id: 19e80f4f-236a-46dd-8e5f-e35cc489a591
Report Status: 268566528
Hashed bucket: 38dd2bbe130a69d3fbdad4dc94ea3d82
Cab Guid: 0 }

**Comment by TheRealBrainbug** [comment]
Exact same here.

Running on 64 bit Windows with 32698 MB RAM
Game Version: v1.18.15 (Stable)
04.12.2023 13:34:43: Critical error occurred
Loaded Mods: game@1.18.15, creative@1.18.15, survival@1.18.15
System.InvalidOperationException: Stack empty.
   at System.Collections.Generic.Stack`1.ThrowForEmptyStack()
   at System.Collections.Generic.Stack`1.Peek()
   at Vintagestory.GameContent.GuiDialogHandbook.ReloadPage() in VSSurvivalMod\Systems\Handbook\Gui\GuiDialogHandbook.cs:line 256
   at Vintagestory.GameContent.ModSystemTutorial.reloadTutorialPage() in VSSurvivalMod\Systems\Handbook\Tutorial\ModSystemTutorial.cs:line 267
   at Vintagestory.GameContent.ModSystemTutorial.onStateUpdate(ActionBoolReturn`1 stepCall) in VSSurvivalMod\Systems\Handbook\Tutorial\ModSystemTutorial.cs:line 241
   at Vintagestory.Client.NoObf.InputAPI.TriggerInWorldAction(EnumEntityAction action, Boolean on, EnumHandling& handling) in VintagestoryLib\Client\API\InputAPI.cs:line 42
   at Vintagestory.API.Common.EntityControls.AttemptToggleAction(EnumEntityAction action, Boolean on) in VintagestoryApi\Common\Entity\EntityControls.cs:line 337
   at Vintagestory.Client.NoObf.SystemPlayerControl.OnGameTick(Single dt) in VintagestoryLib\Client\Systems\Player\PlayerControl.cs:line 68
   at Vintagestory.Common.EventManager.TriggerGameTick(Int64 ellapsedMilliseconds, IWorldAccessor world) in VintagestoryLib\Common\EventManager.cs:line 59
   at Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt) in VintagestoryLib\Client\ClientMain.cs:line 796
   at Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime) in VintagestoryLib\Client\ClientMain.cs:line 712
   at Vintagestory.Client.GuiScreenRunningGame.RenderToPrimary(Single dt) in VintagestoryLib\Client\MainMenu\Screens\GuiScreenRunningGame.cs:line 200
   at Vintagestory.Client.ScreenManager.Render(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 668
   at Vintagestory.Client.ScreenManager.OnNewFrame(Single dt) in VintagestoryLib\Client\ScreenManager.cs:line 643
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(FrameEventArgs e) in VintagestoryLib\Client\ClientPlatform\GameWindow.cs:line 77
   at OpenTK.Windowing.Desktop.GameWindow.Run()
   at Vintagestory.Client.ClientProgram.Start(ClientProgramArgs args, String[] rawArgs) in VintagestoryLib\Client\ClientProgram.cs:line 313
   at Vintagestory.Client.ClientProgram.<>c__DisplayClass9_0.<.ctor>b__1() in VintagestoryLib\Client\ClientProgram.cs:line 129
   at Vintagestory.ClientNative.CrashReporter.Start(ThreadStart start) in VintagestoryLib\Client\ClientPlatform\ClientNative\CrashReporter.cs:line 93

**Comment by radfast** [comment]
Fixed in next upcoming 1.19-pre release, many thanks to @korobya
